dev_guide/deployments/kubernetes_deployments.adoc

```diff
@@ -0,0 +1,187 @@
+1.4,
+endif::[]
+ifdef::openshift-enterprise[]
+3.4,
+endif::[]
+support for Kubernetes deployments has been added as a
+ifdef::openshift-origin[]
+link:https://github.com/openshift/origin#alpha-and-unsupported-kubernetes-features[Tech Preview]
+endif::[]
+ifdef::openshift-enterprise[]
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+endif::[]
+feature.
+
+Like deployment configurations, Kubernetes deployments describe the desired
+state of a particular component of an application as a pod template. Kubernetes
+deployments create _replica sets_ (an iteration of
+xref:../../architecture/core_concepts/deployments.adoc#replication-controllers[replication controllers]), which orchestrate pod lifecycles.
+
+For example, an example Kubernetes deployment that creates a replica set to
+bring up three nginx pods:
+
+.Example Kubernetes Deployment Definition *_nginx-deployment.yaml_*
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.7.9
+        ports:
+        - containerPort: 80
+----
+
+You can use the CLI to operate on Kubernetes deployments and replica sets like
+other object types, as described in
+xref:../../cli_reference/basic_cli_operations.adoc#oc-common-operations[Common Operations], like `create`, `get`, and `describe`. Use `deployments` for
+Kubernetes deployments and `replicasets` or `rs` for replica sets as the object
+type.
+
+See the Kubernetes documentation for more details about
+link:http://kubernetes.io/docs/user-guide/deployments/[Deployments] and
+link:http://kubernetes.io/docs/user-guide/replicasets/[Replica Sets],
+substituting `oc` for `kubectl` in CLI usage examples.
+
+[[deployments-vs-deployment-configurations]]
+== Deployments vs Deployment Configurations
+
+Because deployment configurations existed in {product-title} prior to
+deployments being added in Kubernetes 1.2, the latter object type naturally
+diverges slightly from the former. The long-term goal in {product-title} is to reach
+full feature parity in Kubernetes deployments and switch to using them as a
+single object type that provides fine-grained management over applications.
+
+Kubernetes deployments are supported to ensure upstream projects and examples
+that use the new object type can run smoothly on {product-title}. Given the
+current feature set of Kubernetes deployments, you may want to use them instead
+of deployment configurations in {product-title} if you do not plan to use any of
+the following in particular:
+
+- xref:../../dev_guide/managing_images.adoc#dev-guide-managing-images[image streams]
+- xref:../../dev_guide/deployments/deployment_strategies.html#lifecycle-hooks[lifecycle hooks]
+- xref:../../dev_guide/deployments/deployment_strategies.html#custom-strategy[Custom deployment strategies]
```

Comment:
I capitalized it since we capitalize it in the doc it links to, as the type of strategy. I could go either way here.